### PR TITLE
[JS] Topic: Enhanced JSON Documentation

### PR DESCRIPTION
**Pull request by rwiteshbera**
<hr>
Fixes issue #1882 
## Description 📜

Topics Covered
1. What is JSON
2. Structure of JSON
3. Why should we use JSON?
4. How to access JSON Data?
5. JSON values 
6. Arrays in JSON (Onedimensional/  Mulidimensional Array)
7. JSON vs XML
8. Delete JSON key/Value Pairs
9. JSON Stringify and Parse
10. Looping through JSON Array Items

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] JavaScript

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi @s4-2 Please review.

**Review by s4-2 — APPROVED** [review]
@rwiteshbera Very well formated, looks good to me:)

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
I think that it seems like you have just replaced the existing README. Am I right?

You are supposed to add your own points to it without removing the existing documentation. Please revert to the previous file, and improve and add your points to it again. You can do this in this same PR if you can revert to Aniket's README and start afresh, but if you are not able to, you can make a new PR.

**Comment by rwiteshbera** [comment]
@paramsiddharth Sir, I have modified Aniket's README.md and added topics on JSON like
- Arrays in JSON (Onedimensional/ Multidimensional Array)
- Delete JSON key/Value Pairs
- Looping through JSON Array Items

File: JavaScript/Topics/8.JSON/README.md [hunk 1]
```diff
@@ -1,142 +1,261 @@
...
+  <student>
+    <firstName>Rahul</firstName> <lastName>Roy</lastName>
+  </student>
+  <student>
+    <firstName>Raj</firstName> <lastName>Biswas</lastName>
+  </student>
+  <student>
+    <firstName>Anjali</firstName> <lastName>Mathur</lastName>
+  </student>
+</students>
```
Comment (dakshsethi) [inline JavaScript/Topics/8.JSON/README.md]: Write XML like that

```xml
<students>
    <student>
        <firstName>Rahul</firstName>
        <lastName>Roy</lastName>
    </student>
    <student>
        <firstName>Raj</firstName>
        <lastName>Biswas</lastName>
    </student>
    <student>
        <firstName>Anjali</firstName>
        <lastName>Mathur</lastName>
    </student>
</students>
```

**Review by dakshsethi — CHANGES_REQUESTED** [review]

File: JavaScript/Topics/8.JSON/README.md (hunk 1 above)
Comment (paramsiddharth) [reply JavaScript/Topics/8.JSON/README.md]: @rwiteshbera Look at the suggested changes. :)

**Review by paramsiddharth — APPROVED** [review]
Make the changes mentioned by Daksh. The rest looks good to me!

File: JavaScript/Topics/8.JSON/README.md (hunk 1 above)
Comment (dakshsethi) [reply JavaScript/Topics/8.JSON/README.md]: @rwiteshbera work on these changes!!

**Review by dakshsethi — CHANGES_REQUESTED** [review]

**Comment by rwiteshbera** [comment]
@dakshsethi Sorry Sir for the late, I will do it by today.

**Comment by dakshsethi** [comment]
> @dakshsethi Sorry Sir for the late, I will do it by today.

It's completely okay @rwiteshbera
Whenever u have worked on the changes, just tag me so that I can look at and approve your PR.

**Comment by paramsiddharth** [comment]
@rwiteshbera Make the requested changes and your PR will be good to merge!

**Comment by rwiteshbera** [comment]
@paramsiddharth @dakshsethi  Done Sir! Please review it once.

**Review by dakshsethi — APPROVED** [review]
Looks good to me!! 🔥🔥🔥

**Pull request merged**